Framework/PythonInterface/mantid/api/src/Exports/AlgorithmProperty.cpp

```diff
@@ -39,9 +39,11 @@ AlgorithmProperty *createPropertyWithValidatorAndDirection(const std::string &na
  * when passed to the framework
  * @return A pointer to a new AlgorithmProperty object
  */
-AlgorithmProperty *createPropertyWithValidator(const std::string &name, IValidator *validator) {
-  return createPropertyWithValidatorAndDirection(name, validator, Mantid::Kernel::Direction::Input);
-}
+const std::function<const AlgorithmProperty *(const std::string &, const IValidator *)> createPropertyWithValidator =
```

Comment:
@MohamedAlmaki this is causing build failures any idea why?

Reply:
will check out

Reply:
It appears that Boost::Python is having a problem reading the signature of the function pointer. I will attempt to resolve the error on my Linux machine, and if that is not possible, I may revert the change.